### ISSUE TITLE
Net Worth

### ISSUE THREAD

**Issue by giuseppebaldi** [soloam/ha-fireflyiii-integration#38]
Would it be possible to add net worth as a sensor for HA?

**Comment by soloam** [comment]
Are you talking about the info that is available at the main Dashboard?

**Comment by giuseppebaldi** [comment]
Yes, not sure if that's available through the API.

**Comment by soloam** [comment]
Yes that's possible and it's something I also considered adding! I'll try to add it on the next release! Thank you for the feedback

**Comment by giuseppebaldi** [comment]
Great, thank you!!!

**Comment by Jerry27555** [comment]
Yes, it's possible to add "net worth" as a sensor in Home Assistant by using a RESTful API or a custom sensor that pulls data from a financial source. You would need to configure the sensor in your Home Assistant setup and set it to update at a specified interval.
